renderdoc/driver/gl/gl_manager.cpp

```diff
@@ -608,8 +609,22 @@ void GLResourceManager::PrepareTextureInitialContents(ResourceId liveid, Resourc
 
     if(HasExt[ARB_texture_swizzle] || HasExt[EXT_texture_swizzle])
     {
-      gl.glGetTextureParameterivEXT(res.name, details.curType, eGL_TEXTURE_SWIZZLE_RGBA,
-                                    (GLint *)&state->swizzle[0]);
+      if(IsGLES)
```

Comment:
These individual swizzle parameters exist on desktop GL as well - I would say let's just change to the individual parameters for all cases with a comment saying that the RGBA enum isn't supported on GLES, and ditch the `eGL_TEXTURE_SWIZZLE_RGBA`. Also since it shows up in a few places it would be nice to wrap it into utility functions:

```cpp
void GetTextureSwizzle(GLuint tex, GLenum target, GLint *swizzleRGBA);
void SetTextureSwizzle(GLuint tex, GLenum target, GLint *swizzleRGBA);
```